src/pages/about.js

```diff
@@ -0,0 +1,82 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import { useStaticQuery, graphql } from "gatsby";
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer";
+
+import PageLayout from "../components/PageLayout";
+import TitleArea from "../components/TitleArea";
+import DividerLine from "../components/DividerLine";
+import hackOregonTeam from "../images/team.png";
+import { colors, smBreak } from "../_Theme/UpdatedBrandTheme";
+import Staff from "./About/Staff";
+import FeaturedPost from "./About/FeaturedPost";
+
+const about = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const { contentfulHeading, contentfulCallToActionBlock } = useStaticQuery(
+    graphql`
+      query {
+        contentfulHeading(contentful_id: { eq: "29hwKSc4FX6b1sM8ydG5EX" }) {
+          id
+          title
+          subtitle
+        }
+        contentfulCallToActionBlock(
+          contentful_id: { eq: "3crcjqHYZc2g6Avzs4xENL" }
+        ) {
+          tagline
+          summary {
+            json
+          }
+        }
+      }
+    `
+  );
+
+  return (
+    <PageLayout
+      title={contentfulHeading.title}
+      keywords={[`Civic Software Foundation`, `CIVIC Platform`]}
```

Comment:
You should add swoop info here so that there's a different color and swoop direction under the Staff section. (unless matching is what you're going for)